### PR TITLE
Move UI of issues list closer to iOS visual design

### PR DESCRIPTION
**Pull request by ianyh**
I've done some work on this, and it is probably at the stage where it could use eyes.

### CODE REVIEW

**Comment by subdigital** [comment]
Looking good so far. I noticed a few issues here and there. I think we're close enough to push forward with this implementation.

File: FiveCalls/FiveCalls/IssuesViewController.swift [hunk 1]
```diff
@@ -125,7 +156,16 @@ class IssuesViewController : UITableViewController {
         loadIssues()
     }
 
+    fileprivate func setTitleLabel(location: UserLocation?) {
+        locationButtonItem?.title = location?.locationDisplay ?? "Set Location"
```
Comment (subdigital) [inline FiveCalls/FiveCalls/IssuesViewController.swift]: Need localization for "Set Location" if it doesn't exist already.  (Check `R.strings.localized.`...)

**Comment by subdigital** [comment]
This button color looks like these are disabled. The same color is on the welcome screen.

**Comment by nickoneill** [comment]
It didn't look like those button colors were part of this commit, I changed them to a darker blue in #169

**Comment by ianyh** [comment]
I do plan on finishing this, FYI. I haven't quite had the spoons lately to get it across the line, but I'll hopefully get to it soon.

File: FiveCalls/FiveCalls/IssuesViewController.swift (hunk 1 above)
Comment (ianyh) [inline FiveCalls/FiveCalls/IssuesViewController.swift]: Done. There was already a localizable string for it!

**Comment by ianyh** [comment]
Turns out I had the spoons. I fixed the issue with iPad first launch. I figured I would hold off on rebasing from other changes until the new commits are reviewed, and the relevant PRs are merged.

**Comment by nickoneill** [comment]
Thanks! Working on getting the other items in shortly, I'll check this out as well

**Comment by ianyh** [comment]
Okay, it's been rebased onto master. I think the storyboard changes from #167 are still correct.

**Comment by nickoneill** [comment]
Thanks @ianyh! I'll check this out

**Pull request closed without merging**